includes/Modules/Tag_Manager.php

```diff
@@ -95,6 +96,18 @@ final class Tag_Manager extends Module
 	public function register() {
 		$this->register_scopes_hook();
 
+		add_filter(
+			'googlesitekit_apifetch_preload_paths',
+			function ( $paths ) {
+				$modules_routes = array(
+					// Preload Analytics settings in case it isn't active.
+					'/' . REST_Routes::REST_ROOT . '/modules/analytics/data/settings',
+				);
+
+				return array_merge( $paths, $modules_routes );
+			}
+		);
```

Comment:
Is this needed or just an optimization? I wonder whether we can get along without adding it. How much does making the actual REST API request here affect the perceived speed?

Reply:
Yeah I wouldn't say it's needed, but avoids the extra request for settings during setup. I haven't measured the difference in perceived speed but it's worth comparing. This is in the IB though.

Reply:
This doesn't appear to be doing anything after taking a closer look as the preloaded data is already expired by the time this is requested, I'll remove it.